### PR TITLE
S4t1 - front end 

### PR DESCRIPTION
**Pull request by jesshamby**

### CODE REVIEW

File: JS/functions.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            position: marker.markersObject,
+            map,
+            title: marker.name
+        })
+        newMarker.id = marker.id;
+        newMarker.difficulty = marker.difficulty;
+        newMarker.setIcon(markerIcons[newMarker.difficulty]);
+        // Marker Event Listener
+        google.maps.event.addListener(newMarker, "click", function() {
+            displayWalkInfo(newMarker.id);
```
Comment (Roomsycakes01) [inline JS/functions.js:25]: could you use e.currentTarget.id and use a callback?

File: JS/functions.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        "5": 'http://maps.google.com/mapfiles/ms/icons/purple-dot.png'
+    }
+
+    markersArray.forEach(function (marker) {
+        let newMarker = new google.maps.Marker({
+            position: marker.markersObject,
+            map,
+            title: marker.name
+        })
+        newMarker.id = marker.id;
```
Comment (Roomsycakes01) [inline JS/functions.js]: I think you can move these into the new marker brackets e.g id: marker.id, difficulty: marker.difficulty

**Review by Roomsycakes01 — COMMENTED** [review]
big chungus

File: JS/functions.js (hunk 1 above)
Comment (jesshamby) [reply JS/functions.js:25]: Going to leave this for now as this isn't part of this story - think it is s2 :)

**Pull request merged**